### PR TITLE
[4.0.0] Update 4.0.0 REST APIs

### PR DESCRIPTION
**Pull request by msm1992**
## Purpose
This PR will,
* Add new REST APIs introduced with API-M 4.0.0 to publisher, devportal and admin redocly docs
* Update postman collections for  publisher, devportal and admin
* Add exported postman collection resources to doc assets
* Remove references to admin api v0.17 which is removed from 4.0.0

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: en/mkdocs.yml [hunk 1]
```diff
@@ -801,7 +801,6 @@ nav:
                 - Developer Portal API v2: reference/product-apis/devportal-apis/devportal-v2/devportal-v2.md
             - Admin APIs:
                 - Admin APIs v2: reference/product-apis/admin-apis/admin-v2/admin-v2.md
-                - Admin APIs v0.17: reference/product-apis/admin-apis/admin-v0.17/admin-v0.17.md
```
Comment (Mariangela) [inline en/mkdocs.yml:804]: If this is being removed, you need to update page [1] as well.

[1] https://apim.docs.wso2.com/en/latest/reference/product-apis/overview/

**Review by Mariangela — CHANGES_REQUESTED** [review]

**Comment by Mariangela** [comment]
Fixes https://github.com/wso2/docs-apim/issues/3791

File: en/mkdocs.yml (hunk 1 above)
Comment (msm1992) [reply en/mkdocs.yml:804]: Already updated https://github.com/wso2/docs-apim/pull/3970/files#diff-9772aa32fad308a59ad99ee3ca114c0618154183887e27d0b287985ecf02c787L25-L28

Comment (Mariangela) [reply en/mkdocs.yml:804]: Sorry, my bad!

**Review by Mariangela — APPROVED** [review]

**Review by malinthaprasan — APPROVED** [review]

**Pull request merged**